rocfft/PKGBUILD

```diff
@@ -1,32 +1,31 @@
 # Maintainer: Jakub Okoński <jakub@okonski.org>
 # Maintainer: Markus Näther <naetherm@cs.uni-freiburg.de>
 pkgname=rocfft
-_pkgver=3.3.0
-pkgver="$_pkgver"
+pkgver=3.3.0
 pkgrel=1
-pkgdesc="Next generation FFT implementation for ROCm"
+pkgdesc='Next generation FFT implementation for ROCm'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocfft"
```

Comment:
Can we change the url to https://rocm-documentation.readthedocs.io/en/latest/ROCm_Libraries/ROCm_Libraries.html#rocfft?

Reply:
As I wrote before: The official docu is just a collection of the READMEs from the github repos. If we change the url, we should do this for all packages (at once?).

Reply:
Since its not that important lets just tackle the issue as we see it

Reply:
I've updated the URL.